ui/pages/onboarding-flow/recovery-phrase/index.scss

```diff
@@ -62,11 +94,46 @@
     flex-direction: column;
     justify-content: center;
     align-items: center;
+    width: 100%;
+
+    @media screen and (max-width: $break-small) {
+      margin-inline-start: 40px;
+      margin-inline-end: 40px;
+      margin-bottom: 40px;
+    }
+
+    &__confirm {
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+      width: 100%;
+      margin-bottom: 40px;
+
```

Comment:
```suggestion
      margin-bottom: 20px;
 
      @media screen and (min-width: $break-small) {
        margin: 20px 0 0 0;
      }

```

Reply:
This resolves this margin issue: ![Screen Shot 2022-04-13 at 1 13 44 PM](https://user-images.githubusercontent.com/34557516/163244150-d90fb96d-be68-4423-a5ed-6e8718544174.png)